### ISSUE TITLE
Add token as part of the constructor

### ISSUE THREAD

**Issue by alexisgayte** [brianmcmichael/dss-vest#1]
https://github.com/brianmcmichael/dss-vest/blob/f808e8a942305dcc3409f00cf320d06283a761d8/src/DssVest.sol#L9

**Comment by brianmcmichael** [comment]
Yes, this might be better as an `immutable` variable set in the constructor so this can work with kovan. This is easier for governance participants to audit. If the goal is to generalize this for other projects I can see how that might be desirable, however.

**Comment by alexisgayte** [comment]
Your code is not testable if it is hard coded.
That is another point to add ;)

**Comment by brianmcmichael** [comment]
Added tests at 420cfd5e6bbec01f1cecf5d946cfa7b025ce7e34
